src/site/content/en/blog/install-thumbor/index.md

```diff
@@ -0,0 +1,184 @@
+
+
+## Use Thumbor
+
+You’ve successfully installed Thumbor. Next, learn about using Thumbor in the [Use Thumbor to optimize images](https://web.dev/use-thumbor-to-optimize-images) codelab.
+
+If you are using this guide to set up Thumbor in production, you may also want to check out the optional section below on `systemd`. 
+
+
+## Appendix: Configuring Systemd
+
+This step explains how to make sure that the Thumbor process keeps running, even after the VM has been restarted. This step is important for production sites, but optional if you’re just playing around with Thumbor.
+
+[Systemd](https://www.freedesktop.org/software/systemd/man/systemd.html) is the “system and service manager” for Linux operating systems. `systemd` makes it easy to configure when services (processes) run.
+
+You will be configuring `systemd` to automatically start Thumbor on VM boot. If the VM is restarted, the Thumbor process will automatically restart as well. This is much more reliable than relying on user intervention to start Thumbor.
+
+Navigate to the `/lib/systemd/system` directory. This directory contains the service files for `systemd`.
+
+```bash
+cd /lib/systemd/system
+```
+
+As superuser, create a `thumbor.service` file.
+
+```bash
+sudo touch thumbor.service
+```
+
+Using your favorite text editor (vim and nano come pre-installed on Ubuntu or you can install another editor), add the following configuration to `thumbor.service`. This configuration will run `/usr/local/bin/thumbor` (i.e. the Thumbor binary) once networking is available and will restart Thumbor on [failure](https://www.freedesktop.org/software/systemd/man/systemd.service.html#Restart=). 
+
+```
+[Unit]
+
+Description=Service for Thumbor image CDN
+
+Documentation=https://thumbor.readthedocs.io/en/latest/
+
+After=network.target
+
+[Service]
+
+ExecStart=/usr/local/bin/thumbor
+
+Restart=on-failure
+
+[Install]
+
+WantedBy=multi-user.target
+```
+
+`systemctl` is the utility used to manage `systemd`. Use the `start` command to start Thumbor.
+
+```bash
+sudo systemctl start thumbor.service
+```
+Note: If Thumbor is currently running, you should stop it before attempting to start Thumbor using `systemctl`.
+
+Next, “enable” Thumbor. This means that Thumbor will automatically start on boot.
+
+```bash
+sudo systemctl enable thumbor.service
+```
+
+Verify that you’ve successfully configured `systemd` by running the `status` command.
+
+```bash
+systemctl status thumbor.service
+```
+
+If you’ve successfully set up thumbor.service to use `systemd`, the [status](https://www.freedesktop.org/software/systemd/man/systemctl.html#status%20PATTERN%E2%80%A6%7CPID%E2%80%A6%5D) should show that it is enabled and active.
+
+<figure class="w-figure  w-figure--center">
+  <img src="./systemd.jpg" alt="Systemctl displaying the status of Thumbor">
```

Comment:
I don't want to hold up the post but I was wondering if we might want to come back at some point and use a larger version of this image. The max width is ~800px. Currently at this size the image is a little hard to read.

cc @kaycebasques